target/ppc/translate/storage-ctrl-impl.c.inc

```diff
@@ -106,6 +106,20 @@ static bool trans_SLBMFEV(DisasContext *ctx, arg_SLBMFEV *a)
     return true;
 }
 
+static bool trans_SLBMFEE(DisasContext *ctx, arg_SLBMFEE *a)
+{
+    REQUIRE_64BIT(ctx);
+    REQUIRE_INSNS_FLAGS(ctx, SEGMENT_64B);
+    REQUIRE_SV(ctx);
+
+#if !defined(CONFIG_USER_ONLY) && defined(TARGET_PPC64)
+    gen_helper_load_slb_esid(cpu_gpr[a->rt],cpu_env, cpu_gpr[a->rb]);
```

Comment:
ditto

Reply:
Fixed in 6632babcfbf041b0bc0ea23d2de03be9f5c7ca55